include/lsst/ctrl/events/StatusEvent.h

```diff
@@ -63,30 +39,31 @@ namespace events {
 class StatusEvent : public Event
 {
 public:
-    static const std::string ORIGINATORID;
-    static const std::string LOCALID;
-    static const std::string PROCESSID;
-    static const std::string IPID;
+    static const std::string ORIG_HOSTNAME;
+    static const std::string ORIG_PROCESSID;
+    static const std::string ORIG_LOCALID;
 
     StatusEvent();
     virtual ~StatusEvent();
 
     StatusEvent(cms::TextMessage *msg);
-    StatusEvent(const std::string& runid, int64_t originator, const PropertySet& ps);
-    StatusEvent(const std::string& runid, int64_t originator, const PropertySet::Ptr psp);
+
+    StatusEvent(const LocationID& originator, const PropertySet& ps);
+    StatusEvent(const LocationID& originator, const PropertySet& ps, const PropertySet& filterable);
+
+    StatusEvent(const std::string& runid, const LocationID& originator, const PropertySet& ps);
+    StatusEvent(const std::string& runid, const LocationID& originator, const PropertySet& ps, const PropertySet& filterable);
+    StatusEvent(const std::string& runid, const LocationID& originator, const PropertySet::Ptr psp);
+    StatusEvent(const std::string& runid, const LocationID& originator, const PropertySet::Ptr psp, const PropertySet& filterable);
```

Comment:
Having overrides that vary only in whether a parameter is a reference or pointer unnecessarily duplicates code.  If the caller has a pointer, it's easy for him to call by reference, so get rid of the pointer versions.